easybuild/easyconfigs/l/libtool/libtool-2.4.6.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'libtool'
+version = '2.4.6'
+
+homepage = 'http://www.gnu.org/software/libtool'
+description = """GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries
+ behind a consistent, portable interface."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
Done, for this and the other ones.  Should I change the dummy toolchain version in the M4 build file as well?  

What's the purpose of the "version" field in a dummy toolchain, anyway? and why does it matter?